examples/convecting_plankton.jl

```diff
@@ -1,88 +1,135 @@
-# # Convecting plankton
+# # Plankton mixing and blooming
 #
-# In this example, two-dimensional convection into a stratified fluid
-# mixes a phytoplankton-like tracer. This example demonstrates:
+# In this example, we simulate the mixing of phytoplankton by convection
+# that decreases in time and eventually shuts off, thereby precipitating a
+# phytoplankton bloom. A similar scenario was simulated by
+# [Taylor and Ferrari (2011)](https://aslopubs.onlinelibrary.wiley.com/doi/abs/10.4319/lo.2011.56.6.2293),
+# providing evidence that the
+# ["critical turbulence hypothesis"](https://en.wikipedia.org/wiki/Critical_depth#Critical_Turbulence_Hypothesis)
+# explains the explosive bloom of oceanic phytoplankton
+# observed in spring.
 #
-#   * How to set boundary conditions.
-#   * How to insert a user-defined forcing function into a model.
+# The phytoplankton in our model advect, diffuse, grow, and die according to
```

Comment:
is "phytoplankton" plural? I think it reads better with "advects, diffuses, grows, and dies according..."

Reply:
Phytoplankton is plural! "Phytoplankton concentration" is not, but I used phytoplankton intentionally here (you'd be right to point out that the model makes an approximation and strictly speaking applies more directly to phytoplankton concentration). An individual plankton is called a "plankter".